analysis/monkmistweaver/src/modules/spells/EssenceFont.tsx

```diff
@@ -46,6 +61,8 @@ class EssenceFont extends Analyzer {
       Events.refreshbuff.by(SELECTED_PLAYER).spell(SPELLS.ESSENCE_FONT_BUFF),
       this.refreshEssenceFontBuff,
     );
+    this.addEventListener(Events.EndChannel.by(SELECTED_PLAYER), this.handleEndChannel);
```

Comment:
You should be able to do `.spell(SPELLS.ESSENCE_FONT)` after .by so filter for just EF end cast events